commands/members/add.js

```diff
@@ -3,36 +3,76 @@
 let cli = require('heroku-cli-util')
 let co = require('co')
 let extend = require('util')._extend
+let Utils = require('../../lib/utils')
 
 function * run (context, heroku) {
-  // Users receive `You'll be billed monthly for teams over 5 members.`
-  // message when going over the FREE_TEAM_LIMIT
-  const FREE_TEAM_LIMIT = 5
-  let org = context.org
+  let orgInfo = yield Utils.orgInfo(context, heroku)
+  let orgName = context.org
+
+  const warnMembershipLimit = function * (totalMembers) {
+    // Users receive `You'll be billed monthly for teams over 5 members.`
+    if (totalMembers === 6) {
```

Comment:
`FREE_TEAM_LIMIT` was at least a constant. Having to hunt for the number 5 and 6 later on is much more difficult than looking for a constant.